### DISCUSSION TITLE
App renders differently when running as net8.0-windows10.0.19041 and net8.0-desktop

### DISCUSSION THREAD

**Discussion by kucint** [unoplatform/uno#16608] [Q&A] [answered]
I would like to use "Fluent" pattern in my cross-platform app.

I created a test app that supports both `net8.0-windows10.0.19041` and `net8.0-desktop`. 

when running an app under windows as net8.0-windows10.0.19041 and net8.0-desktop, I observe huge differences how both apps are rendered.

 `net8.0-windows10.0.19041`
![image](https://github.com/unoplatform/uno/assets/54866088/0c6dc6a1-b8ff-4d76-9082-4db7b7e50fd7)

 `net8.0-desktop`
![image](https://github.com/unoplatform/uno/assets/54866088/f7508e1b-9386-4742-af98-0d1ca0d99d18)

I expect that `net8.0-desktop` will mimic `net8.0-windows10.0.19041`.
This is not the case: 
 - Windows Theme accent color is not respected
 - scrollbars in `ScrollViewer` are very simple and look not appealing
 - `ListBox` is not scrollable
 - `ListBox` disrespects background color
 - animations don't work

Here you have the Source code: [UnoScrollBarRenderApp.zip](https://github.com/unoplatform/uno/files/15235742/UnoScrollBarRenderApp.zip)

**Question**: Will this discrepancy be be fixed in the future release(s) or do am I missing something?

**Comment by jeromelaban** [comment]
Thanks for the report! 

At this time, `ListBox` is not implemented: https://github.com/unoplatform/uno/issues/1362. You can use `ListView` or `ItemsRepeater` instead.

I do not believe we support accent color at this time, @MartinZikmund may know more about this.

For animations, can you specify which ones?

`ScrollBar` styling is likely something that was changed recently in winui that has not yet followed in Uno, @MartinZikmund may also know.

Finally, in general, it's best to avoid aggregated questions and open multiple discussions or issues, it's easier for us to track.

**Reply by MartinZikmund** [reply, marked as answer]
Indeed, the color differences are caused by the fact that Uno currently only uses the default "windows" blue. If this is a feature you would like to see, please vote on https://github.com/unoplatform/uno/issues/4444

For `ScrollBar` styling we should be able to bring in the latest styles, I will create an issue for this

**Reply by kucint** [reply]
Hi @MartinZikmund , I created an issue for scrollbar render problem [#16718](https://github.com/unoplatform/uno/issues/16718)

**Reply by kucint** [reply]
I close this topic because it has been split into two separated issues: #4444 and #16718.

**Reply by agneszitte** [reply]
https://github.com/unoplatform/uno/issues/16718 is a duplicate and can be tracked in https://github.com/unoplatform/uno/issues/16648
